internal/buildinfo/version.go

```diff
@@ -105,15 +212,3 @@ func parseVesion(v string) semver.Version {
 	}
 	return parsedV
 }
```

Comment:
CI's complaining about `parseVesion` being unused. Seems like everything that uses it got yanked, so it should be safe to remove this too.